lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb

```diff
@@ -13,15 +13,13 @@ Make an appointment at the British embassy in Bogota to swear an affidavit (writ
 
 ## Fees
 
-<%= render partial: 'consular_fees_table_items.govspeak.erb',
-           collection: calculator.services,
-           as: :service,
-           locals: { calculator: calculator } %>
-
+Service | Fee
+- |-
+Affidavit or affirmation for marriage | £50
+ 
 You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for Colombia](/government/publications/colombia-consular-fees).
-
-<%=  render partial: 'how_to_pay.govspeak.erb',
-            locals: {calculator: calculator}%>
+ 
+You can pay by cash or credit card, but not by personal cheque.
```

Comment:
You can remove the `You can pay by cash or credit card, but not by personal cheque.` line as the partial will render that.